pom.xml

```diff
@@ -39,18 +39,6 @@
 		</developer>
 	</developers>
 
-	<contributors>
```

Comment:
You shouldn't remove them, this part has already been fixed. Simply delete this commit.